spring-data-jest/src/test/java/com/github/vanroy/springdata/jest/JestElasticsearchTemplateTests.java

```diff
@@ -132,7 +132,7 @@ public void before() {
 	@Test
 	public void shouldReturnCountForGivenCriteriaQuery() {
 		// given
-		String documentId = randomNumeric(5);
+		String documentId = String.valueOf(ThreadLocalRandom.current().nextLong()).substring(1, 6);
```

Comment:
I really struggle with this one. Spring's `StringUtils` doesn't offer anything similar, so I concocted my own. There is a slight risk of `ThreadLocalRandom.current().nextLong()` returning a value with less than 5 digits in it, but it's infinitesimally small.